src/ui/components/AppWrapper/AppWrapper.tsx

```diff
@@ -187,7 +192,7 @@ const AppWrapper = (props: { children: ReactNode }) => {
       };
       handleMessages();
     }
-  }, [isOnline]);
+  }, [isOnline, authentication, dispatch]);
```

Comment:
Not sure why `dispatch` is in this list. Also, can it be as specific as `authentication.loggedIn` or no?

Reply:
React Hook throws a warning like this if we don't pass `dispatch` here
"React Hook useEffect has a missing dependency: 'dispatch'. Either include it or remove the dependency array.eslint[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)".

Reply:
ok and what about `authentication` vs `authentication.loggedIn`?

Reply:
Yes, we can use `authentication.loggedIn`